internal/commands/user/revoke.go

```diff
@@ -0,0 +1,151 @@
+	)
+	fs.VarP(&cmd.inputs.State, flagState, flagStateShort, flagStateUsage)
+	fs.BoolVarP(&cmd.inputs.Pending, flagPending, flagPendingShort, false, flagPendingUsage)
+}
+
+// Inputs is the command inputs
+func (cmd *CommandRevoke) Inputs() cli.InputResolver {
+	return &cmd.inputs
+}
+
+// Setup is the command setup
+func (cmd *CommandRevoke) Setup(profile *cli.Profile, ui terminal.UI) error {
+	cmd.realmClient = profile.RealmAuthClient()
+	return nil
+}
+
+// Handler is the command handler
+func (cmd *CommandRevoke) Handler(profile *cli.Profile, ui terminal.UI) error {
+	app, appErr := app.Resolve(ui, cmd.realmClient, cmd.inputs.Filter())
+	if appErr != nil {
+		return appErr
+	}
+	users, usersErr := cmd.inputs.ResolveUsers(ui, cmd.realmClient, app)
+	if usersErr != nil {
+		return usersErr
+	}
+	for _, user := range users {
+		err := cmd.realmClient.RevokeUserSessions(app.GroupID, app.ID, user.ID)
+		cmd.outputs = append(cmd.outputs, userOutput{user: user, err: err})
+	}
+	return nil
+}
+
+// Feedback is the command feedback
+func (cmd *CommandRevoke) Feedback(profile *cli.Profile, ui terminal.UI) error {
+	if len(cmd.outputs) == 0 {
+		return ui.Print(terminal.NewTextLog("No users to revoke, try changing the --user input"))
+	}
+	var outputsByProviderType = map[realm.AuthProviderType][]userOutput{}
+	for _, output := range cmd.outputs {
+		for _, identity := range output.user.Identities {
+			outputsByProviderType[identity.ProviderType] = append(outputsByProviderType[identity.ProviderType], output)
+		}
+	}
+	logs := make([]terminal.Log, 0, len(outputsByProviderType))
+	for _, apt := range realm.ValidAuthProviderTypes {
+		outputs := outputsByProviderType[apt]
+		if len(outputs) == 0 {
+			continue
+		}
+		sort.SliceStable(outputs, getUserOutputComparerBySuccess(outputs))
+		logs = append(logs, terminal.NewTableLog(
+			fmt.Sprintf("Provider type: %s", apt.Display()),
+			userRevokeTableHeaders(apt),
+			userRevokeTableRows(apt, outputs)...,
+		))
+	}
+	return ui.Print(logs...)
+}
+
+func userRevokeTableHeaders(authProviderType realm.AuthProviderType) []string {
+	var headers []string
+	switch authProviderType {
+	case realm.AuthProviderTypeAPIKey:
+		headers = append(headers, headerName)
+	case realm.AuthProviderTypeUserPassword:
+		headers = append(headers, headerEmail)
+	}
+	headers = append(
+		headers,
+		headerID,
+		headerType,
+		headerRevoked,
+		headerDetails,
+	)
+	return headers
+}
+
+func userRevokeTableRows(authProviderType realm.AuthProviderType, outputs []userOutput) []map[string]interface{} {
+	rows := make([]map[string]interface{}, 0, len(outputs))
+	for _, output := range outputs {
+		rows = append(rows, userRevokeTableRow(authProviderType, output))
+	}
+	return rows
+}
+
+func userRevokeTableRow(authProviderType realm.AuthProviderType, output userOutput) map[string]interface{} {
+	var details string
+	if output.err != nil {
+		details = output.err.Error()
+	}
+	success := "no"
+	if output.err == nil {
+		success = "yes"
+	}
```

Comment:
Thoughts on outputting yes/no instead of true/false in the cli feedback? I felt it would make for sense to the user
<img width="581" alt="Screen Shot 2021-01-27 at 10 37 54 PM" src="https://user-images.githubusercontent.com/66638373/106090647-4de1e700-60f0-11eb-9c74-165172042ad4.png">

Reply:
I personally prefer it after seeing it, but I don't know if there are any norms we have for displaying information

Reply:
I was thinking the same, I will let @makesitgo chime in here as well

Reply:
I like the look and feel of this as well... do you think this might be something we want for _all_ booleans?  im inclined to think yes, and if so, we could probably make this change in `terminal/table.go`'s `parseValue` function.

what do you all think?

Reply:
I tested out the change in table.go in this PR (seemed to only affect the user commands). I liked the result